apps/worker/src/app/workflow/services/active-jobs-metric.service.ts

```diff
@@ -116,21 +65,4 @@ export class ActiveJobsMetricService {
       });
     };
   }
-
-  public async gracefulShutdown(): Promise<void> {
```

Comment:
moved to scheduler service, in 1 place we handle the shutdown of all schedulers.